app/__init__.py

```diff
@@ -3,21 +3,56 @@
 Copyright (c) 2020 - DevsBranch
 """
 
+import os
 from flask import Flask
+from celery import Celery
 from flask_marshmallow import Marshmallow
 from flask_login import LoginManager
 from flask_sqlalchemy import SQLAlchemy
 from importlib import import_module
 from flask_migrate import Migrate
 from flask_jwt_extended import JWTManager
 from decouple import config as db_config
+from config import config_dict
+
+CELERY_TASK_LIST = [
+    "app.celery_utils",
+]
+
+
+def make_celery(app):
```

Comment:
perhaps we could name this as `celery_init` or `init_celery` ?
that way  it will correlate more closely to what the function does which is initializing celery